llvm/lib/Target/NVPTX/NVPTXISelDAGToDAG.cpp

```diff
@@ -915,6 +920,42 @@ getOperationOrderings(MemSDNode *N, const NVPTXSubtarget *Subtarget) {
 
 } // namespace
 
+NVPTX::Scope NVPTXDAGToDAGISel::getOperationScope(MemSDNode *N,
+                                                  NVPTX::Ordering Ord) const {
```

Comment:
Nit: `O` or `Ordering` would work a bit better here, IMO. I'd probably use just `O` to match `N`.

Reply:
Done